SpotifyAPI/ProxyConfig.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Net;
+
+namespace SpotifyAPI
+{
+    public class ProxyConfig
+    {
+        public string Host { get; set; }
+
+        public int Port { get; set; } = 80;
+
+        public string Username { get; set; }
+
+        public string Password { get; set; }
+
+        /// <summary>
+        /// Whether to bypass the proxy server for local addresses.
+        /// </summary>
+        public bool BypassProxyOnLocal { get; set; }
+
+        public void Set(ProxyConfig proxyConfig)
+        {
+            this.Host = proxyConfig?.Host;
```

Comment:
Please remove redundant `this.`

Reply:
Ops I though I removed them all. My Visual Studio/Resharper/CodeMaid adds them automatically sometimes.